CHANGELOG.md

```diff
@@ -78,6 +78,8 @@ For older changes see the [archived Singularity change log](https://github.com/a
   non-default cgroupfs manager, and no device rules.
 - Ensure `DOCKER_HOST` is honored in non-build flows.
 - Remove duplicated group ids.
+- Refine the misleading message trying to use encrypted filesystem
+  in unprivileged mode.
```

Comment:
There's no need for a changelog message since it pertains to another change that is already in the log and not yet released.

Reply:
resolved bb0a3b5c547a0bf5c062437bf2c4e23202f45fbf